musonant/questioner
# Clean up app authentication

#### What does this PR do?
Clean up app authentication

#### Description of Task to be completed?
- Delete .DS_Store files
- Extend token expiration
- Change response format from array to object

#### How should this be manually tested?
- after cloning the repository
- start the server using `npm run start`
- access the login/signup route, providing the necessary details
- the user's details should be returned, alongside the token in an object

#### Any background context you want to provide?
- NA

#### What are the relevant pivotal tracker stories?
- NA
